### DISCUSSION TITLE
AlDente won't launch on M3 Pro Macbook Pro (Sonoma 14.1)

### DISCUSSION THREAD

**Discussion by Howsezb** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1089] [Q&A]
I download the app, drag to apps and try to launch and nothing happens. No process shown in activity monitor.
Is this a known bug? Really wanted to check out the free version before purchasing

**Comment by MatthiasKerbl** [comment]
Hi @Howsezb,

A new update with M3 MacBook support will be released soon.

Best,
Matthias

**Reply by JeffreyWardman** [reply]
@Howsezb It's been released: https://github.com/AppHouseKitchen/AlDente-Charge-Limiter/releases/tag/1.23
